src/shared/components/mutationTable/column/clonal/ClonalElement.tsx

```diff
@@ -64,8 +66,14 @@ const ClonalCircle: React.FunctionComponent<{
             <circle
                 cx={5}
                 cy={5}
-                r={5}
-                fill={getClonalColor(props.clonalValue)}
+                r={4}
+                stroke={getClonalColor(props.clonalValue)}
+                stroke-width={1}
+                fill={
+                    props.clonalValue !== 'subclonal'
+                        ? getClonalColor(props.clonalValue)
+                        : ClonalColor.WHITE
```

Comment:
Shouldn't this logic be included in getClonalColor?

Reply:
👍  I will create two new functions one for getFillColor and one for getStrokeColor